src/WebUI/Components/KubeSummary.scss

```diff
@@ -8,6 +8,7 @@
 }
 
 .kubernetes-container {
+    width: 100%;

```

Comment:
we had some issues with this, so did not include this, would you check this in webapp also?